image.nix

```diff
@@ -0,0 +1,17 @@
+{ pkgs, app, name, version, ... }:
+pkgs.dockerTools.buildImage {
```

Comment:
how is docker used in this repo? We are not building any containers here. Is it required here?

Reply:
it is not, just leaving as example. we may remove if you think that is a problem.